qa/rpc-tests/mempool_nu_activation.py

```diff
@@ -62,7 +62,7 @@ def run_test(self):
         node1_taddr = get_coinbase_address(self.nodes[1])
         node0_zaddr = self.nodes[0].z_getnewaddress('sapling')
         recipients = [{'address': node0_zaddr, 'amount': Decimal('10')}]
-        myopid = self.nodes[1].z_sendmany(node1_taddr, recipients, 1, 0)
+        myopid = self.nodes[1].z_sendmany(node1_taddr, recipients, 1, 0, 'AllowRevealedSenders')
```

Comment:
I hadn’t made this change because it didn’t seem like it should be necessary, so I’m afraid it’s hiding a bug.

This test has `-allowdeprecated=legacy_privacy`, and there are no UAs involved in the transaction, so shouldn’t the default policy here be `AllowFullyTransparent`? Plus, since it’s a `t->z` tx, it should have required at least `AllowRevealedSenders` before this PR, so I’m not sure why it’s not defaulting to `AllowFullyTransparent` when it appeared to be before.